tests/Features/Methods/ModelExtension.php

```diff
@@ -13,13 +13,18 @@
 class ModelExtension
 {
     /**
-     * @return \Illuminate\Database\Eloquent\Collection<User>
+     * @return \Illuminate\Support\Collection<User>
```

Comment:
It was correct before. `all` returns `\Illuminate\Database\Eloquent\Collection` not ` \Illuminate\Support\Collection`